packages/legend-application-studio/src/components/editor/edit-panel/mapping-editor/InstanceSetImplementationSourceSelectorModal.tsx

```diff
@@ -205,6 +206,7 @@ export const InstanceSetImplementationSourceSelectorModal = observer(
             placeholder={`Select a source...`}
             isClearable={true}
             filterOption={filterOption}
+            formatOptionLabel={getPackageableElementOptionFormatter({})}
```

Comment:
we need to modify this slightly. This won't work well afaik because instance set implementation source selector allows more stuffs than just elements